app/controllers/application_controller.rb

```diff
@@ -1,2 +1,16 @@
 class ApplicationController < ActionController::Base
+    before_action :configure_permitted_parameters, if:  :devise_controller?
```

Comment:
By rails style guides we use 2 spaces instead of 4. To avoid thoughts about styles we can use https://github.com/rubocop/rubocop gem. It's a helpful tool that analyzes your code and finds out some bad styles. I think you can add this gem to your project in separate PR. 

Also, you can read this rails styles guides https://rubystyle.guide/#spaces-indentation
```suggestion
  before_action :configure_permitted_parameters, if:  :devise_controller?
```